src/main/features/response/form/models/statement-of-means/unemploymentDetails.ts

```diff
@@ -1,17 +1,28 @@
-import { IsDefined, IsInt, Min } from 'class-validator'
+import { IsDefined, IsInt, Max, Min } from 'class-validator'
 import { toNumberOrUndefined } from 'common/utils/numericUtils'
 import { ValidationErrors as GlobalValidationErrors } from 'forms/validation/validationErrors'
 
+export class ValidationErrors {
+  static readonly TOO_MANY: string = 'Enter a number between 0 and $constraint1'
```

Comment:
This constant's message will be used when the user enters a negative number as well. What do you think about renaming to just `INVALID_NUMBER`?